### ISSUE TITLE
Error with Settings for Vagrant Setup

### ISSUE THREAD

**Issue by brittainhard** [memex-explorer/memex-explorer#501]
Got this error while I was running the setup

```
==> default: ln: 
==> default: failed to create symbolic link ‘/vagrant/source/memex/settings.py’
==> default: : File exists
==> default: /vagrant/deploy/deploy_vagrant.sh: line 22: /home/vagrant/envs/memex/bin/python: No such file or directory
==> default: /vagrant/deploy/deploy_vagrant.sh: line 24: /home/vagrant/envs/memex/bin/python: No such file or directory
==> default: /vagrant/deploy/deploy_vagrant.sh: line 25: /home/vagrant/envs/memex/bin/python: No such file or directory
==> default: /vagrant/deploy/deploy_vagrant.sh: line 26: /home/vagrant/envs/memex/bin/python: No such file or directory
==> default: modprobe: FATAL: Module aufs not found.
```
and a second one, which I think are related:
```
==> default: Traceback (most recent call last):
==> default:   File "./manage.py", line 8, in <module>
==> default:     
==> default: from django.core.management import execute_from_command_line
==> default: ImportError
==> default: : 
==> default: No module named django.core.management
```

**Comment by ahmadia** [comment]
> ==> default: /vagrant/deploy/deploy_vagrant.sh: line 22: /home/vagrant/envs/memex/bin/python: No such file or directory

Oooph, something has gone very wrong there.  Hold off until I've had a chance to debug this.

**Comment by ahmadia** [comment]
Okay, I've pushed a complete overhaul into master.  Feel free to test, see https://github.com/memex-explorer/memex-explorer/pull/504

**Comment by brittainhard** [comment]
Closing this, was fixed in #504, as far as I can tell.
